demo.html

```diff
@@ -358,5 +363,12 @@ <h3>Effects</h3>
 			Made with hands by <a aria-label="@chinchang457" class="hint--top" href="https://twitter.com/chinchang457">Kushagra Gour</a> in India
 		</div>
 
+		<script>
+			$(window).on('load',function(){
```

Comment:
this is jquery syntax. this page doesn't have jquery, so it wont work. You had to do things in vanilla JS for this page.